README.md

```diff
@@ -94,16 +93,17 @@ The task management app Things stores all your to-dos in a SQLite database file
 
 Here's the terminology used involving the database:
 
-- Area
-- Tag
-- Task
+- area
+- tag
+- Task (capitalized)
```

Comment:
What about using another term here instead of `Task`? Like `item`, `entity`, `todo`, ...?

Reply:
Yes, this is confusing. You're right.

Since the table is called TMTask, how about keeping _task_ as the collective term, but then separating tasks into the types:
- to-do
- project
- heading
?

In the App these same terms are used. I haven't seen the term "task" mentioned anywhere so far in the app, but they do use the term "item" which refers to (to-do, project, heading, _and_ checklist item).